Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM node:6
+FROM node:6-alpine
 
 WORKDIR /opt
 EXPOSE 1337
 VOLUME ["/opt/data"]
 
+RUN apk add --no-cache make gcc g++ python
```

Comment:
These packages need to be removed otherwise the image will be larger than needed.

Add after `npm cache clean`

```
RUN apk del make gcc g++ python
```